essentials-framework/Essentials DM/Essentials_DM/Endpoints/Transmitters/DmTx200Controller.cs

```diff
@@ -127,6 +129,16 @@ public DmTx200Controller(string key, string name, DmTx200C2G tx)
 
             HdcpSupportCapability = eHdcpCapabilityType.HdcpAutoSupport;
 
+            HdmiVideoSyncFeedback = new BoolFeedback(() =>
```

Comment:
for future reference, this can be written simpler:
`HdmiVideoSyncFeedback = new BoolFeedback(() => tx.HdmiInput.SyncDetectedFeedback.BoolValue)`

Lambdas without the `{}` have return implied in them